<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>794</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>13</y>
      <width>1081</width>
      <height>721</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>33</y>
       <width>171</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Current position</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="x_pos_doubleSpinBox">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="y_pos_doubleSpinBox">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="0" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>193</x>
       <y>33</y>
       <width>226</width>
       <height>140</height>
      </rect>
     </property>
     <property name="title">
      <string>Spectrometer settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="scans_to_avg_spinBox"/>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="intg_time_doubleSpinBox"/>
        </item>
        <item row="0" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>Intg. time (ms)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>Scans to average</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="correct_dark_counts_checkBox">
        <property name="text">
         <string>Correct dark counts</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>170</y>
       <width>591</width>
       <height>144</height>
      </rect>
     </property>
     <property name="title">
      <string>Scan settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="x_start_doubleSpinBox">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="value">
           <double>50.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="y_start_doubleSpinBox">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="value">
           <double>50.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Y Start (um)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Y Scan Size (um)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Y Step (um)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QDoubleSpinBox" name="y_step_doubleSpinBox">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="value">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="4" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>X Step (um)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>X Scan Size (um)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>X Start (um)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QDoubleSpinBox" name="x_size_doubleSpinBox">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="value">
           <double>5.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QDoubleSpinBox" name="y_size_doubleSpinBox">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="value">
           <double>5.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QDoubleSpinBox" name="x_step_doubleSpinBox">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
          <property name="value">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Scan progress</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="start_scan_pushButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>60</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Start scan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="interrupt_scan_pushButton">
     <property name="geometry">
      <rect>
       <x>428</x>
       <y>120</y>
       <width>127</width>
       <height>34</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Interrupt scan</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="plot_groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>561</width>
       <height>400</height>
      </rect>
     </property>
     <property name="title">
      <string>Plot</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4"/>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>170</y>
       <width>421</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Move stage to selected position</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>160</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Selected y</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Selected x</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="x_clicked_doubleSpinBox"/>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="y_clicked_doubleSpinBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="move_to_selected_pushButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>61</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Move</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="lock_position_checkBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Lock position</string>
      </property>
     </widget>
     <widget class="QPushButton" name="export_positions_pushButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>90</y>
        <width>131</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Export positions</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="save_positions_checkBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Save positions</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="details_groupBox">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>40</y>
       <width>421</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Details</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="stage_groupBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>330</y>
      <width>481</width>
      <height>400</height>
     </rect>
    </property>
    <property name="title">
     <string>Stage</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1112</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>intg_time_doubleSpinBox</tabstop>
  <tabstop>scans_to_avg_spinBox</tabstop>
  <tabstop>correct_dark_counts_checkBox</tabstop>
  <tabstop>x_start_doubleSpinBox</tabstop>
  <tabstop>y_start_doubleSpinBox</tabstop>
  <tabstop>x_size_doubleSpinBox</tabstop>
  <tabstop>y_size_doubleSpinBox</tabstop>
  <tabstop>x_step_doubleSpinBox</tabstop>
  <tabstop>y_step_doubleSpinBox</tabstop>
  <tabstop>start_scan_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
